test/firtool/w0mem.fir

```diff
@@ -0,0 +1,19 @@
+; RUN: firtool %s --format=fir              | circt-opt | FileCheck %s 
+
+circuit foo :
+
```

Comment:
Could you move this into the test/firparser directory?  You can probably merge it into an existing test as well.